pytrip/ctx.py

```diff
@@ -85,8 +85,9 @@ def create_dicom(self):
             ds.RescaleSlope = 1.0
             ds.PixelRepresentation = 1
             ds.ImagePositionPatient = ["%.3f" % (self.xoffset * self.pixel_size),
-                                       "%.3f" % (self.yoffset * self.pixel_size), "%.3f" % (self.slice_pos[i])]
-            ds.SOPClassUID = '1.2.840.10008.5.1.4.1.1.2'
+                                       "%.3f" % (self.yoffset * self.pixel_size),
+                                       "%.3f" % (self.slice_pos[i])]
+            ds.SOPClassUID = '1.2.840.10008.5.1.4.1.1.2'  # CT Image Storage SOP Class
```

Comment:
Yes, this is a reason why I've added a comment.
Number '1.2.840.10008.5.1.4.1.1.2' corresponds to `CT Image Storage SOP Class` in DICOM terminology.